### DISCUSSION TITLE
Configured Theme colors, slider is not changing its primary color in dark mode only in white mode. Need help!

### DISCUSSION THREAD

**Discussion by JenuelDev** [tusen-ai/naive-ui#1373] [Q&A] [answered]
Configured Theme colors, slider is not changing its primary color in dark mode only in white mode. Need help!

I set the `primaryColor` to `yellowish orange`. but it seems the slider is not following the primaryColor in dark mode.

This is the dark mode did not follow the primaryColor.
![image](https://user-images.githubusercontent.com/31676163/137451184-48efc77f-fc1e-421d-ad3a-3c90b90fef32.png)

this is the light mode, it follows the primaryColor:
![image](https://user-images.githubusercontent.com/31676163/137451302-48c508fb-bda2-410f-a814-eb12185a4ebd.png)

**Comment by 07akioni** [comment, marked as answer]
primaryColorSuppl

**Reply by 07akioni** [reply]
Suppl means supplementary.

Since primary color is too light for slider rail background.

**Reply by JenuelDev** [reply]
Oh thanks, it works now... :)

**Reply by 07akioni** [reply]
Sometimes it's very hard to make two design tokens have same behavior in different themes, since every color has its scene.

**Reply by JenuelDev** [reply]
agree, at least we have controls over the theme for this, which is really great 😁
